Header header

uint8  station_num
uint16  data